sphinxcontrib/inmanta/config.py

```diff
@@ -69,6 +69,11 @@ def _format_group(group_name, opt_list):
             default = str(default).strip()
             yield _indent(":Default: %s" % default)
 
+        environment_variable = opt.get_environment_variable()
+        if environment_variable:
```

Comment:
In which situation is this None or an empty string?

Reply:
Should never happen, I removed the redundant check.